dhall-json/dhall-json.cabal

```diff
@@ -49,7 +49,6 @@ Library
         containers                                     ,
         dhall                     >= 1.22.0   && < 1.25,
```

Comment:
This bound's wrong now, since Dhall.Optics was new in 1.24 (I think).

Reply:
Thanks! Fixed!